### DISCUSSION TITLE
how can I run APIs from golang terminal

### DISCUSSION THREAD

**Discussion by isra270** [ory/hydra#2317] [Q&A] [answered]
I want to debug the APIs in Hydra like create client on `Golang terminal` not show it from `docker log` 

for example  when i run `http://127.0.0.1:4445/clients`  on postman success inserted  I show insert data in postgres on docker .

I want to show it from golang terminal .
I ran go run main.go but it only execute commands .
I am compile hydra on binary that I ran it but I can't reach to API from terminal .

any way can I reach API from postman that I show log on golang terminal

**Comment by k9ert** [comment, marked as answer]
I'm not sure i understand your issue. What do you mean by "Golang terminal" ? Logs are always coming out of stdout. That's a pretty standard way of dealing with logs.

Maybe you can explain what you want to achieve here from a more toplevel point of view? 

Edit:
maybe i got it now. If you run kratos from the commandline, logs should come out od stdout. Should be the same in windows. Here is the most trivial example:
```
➜  kratos git:(83aedcb8) go run main.go serve 
WARN[2021-01-22T16:58:07+01:00] No config file was defined and no file was found in $HOME/.kratos.yaml  audience=application error=map[message:Config File ".kratos" Not Found in "[/home/kim]"] service_name=kratos service_version=
ERRO[2021-01-22T16:58:07+01:00] The provided configuration is invalid and could not be loaded. Check the output below to understand why.  audience=application config_file= service_name=kratos service_version=

identity: <nil>
          ^-- one or more required properties are missing

courier.smtp.connection_uri: <nil>
                             ^-- one or more required properties are missing

selfservice.default_browser_return_url: <nil>
                                        ^-- one or more required properties are missing

FATA[2021-01-22T16:58:07+01:00] The services failed to start because the configuration is invalid. Check the output above for more details.  audience=application service_name=kratos service_version=
exit status 1
➜  kratos git:(83aedcb8)
```

That should work for you?!

**Reply by isra270** [reply]
Thanks so much @k9ert .
